os/arch/arm/src/imxrt/imxrt_flash.h

```diff
@@ -61,9 +61,69 @@
 #include <tinyara/progmem.h>
 
 #include "chip.h"
+#include "imxrt_config.h"
+
+#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#include "chip/imxrt102x_config.h"
+#elif defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT105x)
+#include "chip/imxrt105x_config.h"
+#else
+#error Unrecognized i.MX RT architecture
+#endif
 
 #ifndef __ASSEMBLY__
 
+#if defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
+#define IMXRT_FLEXSPI FLEXSPI
+#define IMXRT_FLASH_SIZE 0x2000 /* 64Mb/KByte */
+#define IMXRT_FLEXSPI_AMBA_BASE FlexSPI_AMBA_BASE
+#define IMXRT_FLASH_PAGE_SIZE 256
+#define IMXRT_SECTOR 0
+#define IMXRT_FLASH_BASE  0x60000000
+#define IMXRT_SECTOR_SIZE 0x1000 /* 4K */
+#define IMXRT_BLOCK_SIZE 0x8000
+#define IMXRT_FLASH_TOTAL_SIZE (IMXRT_FLASH_SIZE*1024)
```

Comment:
Please give spase before and after ' * '

IMXRT_FLASH_SIZE * 1024

Reply:
I fixed it